packages/registry/src/fetch/operator.fetch.ts

```diff
@@ -37,21 +37,21 @@ export class RegistryOperatorFetchService {
       name,
       active,
       rewardAddress,
-      stakingLimit,
-      stoppedValidators,
-      totalSigningKeys,
-      usedSigningKeys,
+      totalVettedValidators,
+      totalExitedValidators,
+      totalAddedValidators,
+      totalDepositedValidators,
     } = operator;
 
     return {
```

Comment:
I'm thinking of refactor this module soon to use new method `getSigningKeys` to get keys in batches directly from the contract. And it may be a good opportunity to migrate to new interfaces

Reply:
Ok